pipeline.py

```diff
@@ -28,61 +36,116 @@ class MorphologicalToken:
     Stores language params for each processed token
     """
     def __init__(self, original_word, normalized_form):
-        pass
+        self.original_word = original_word
+        self.normalized_form = normalized_form
+        self.mystem_tags = ''
+        self.pymorphy_tags = ''
 
     def __str__(self):
-        return "MorphologicalToken instance here"
+        return self.normalized_form + f'<{self.mystem_tags}>' + f'({self.pymorphy_tags})'
 
 
 class CorpusManager:
     """
     Works with articles and stores them
     """
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path = Path(self.path_to_raw_txt_data)
+        all_files = [str(f_path) for f_path in path.iterdir()]
+        raw_files = {}
+
+        for file in all_files:
```

Comment:
there should be a better way to retrieve only files by the mask, take a look at the pathlib documentation